src/sentry/search/snuba/executors.py

```diff
@@ -453,6 +465,25 @@ def trend_aggregation(start: datetime, end: datetime) -> Sequence[str]:
     ]
 
 
+def better_priority_aggregation(
+    start: datetime,
+    end: datetime,
+    aggregate_kwargs: Optional[PrioritySortWeights],
+) -> Sequence[str]:
+    event_age_hours = "divide(now() - timestamp, 3600)"
+    event_score = 1
+    min_score = 0.01
+    max_pow = 16
+    event_halflife_hours = 4  # halves score every 4 hours
+    aggregate_event_score = f"greatest({min_score}, sum(divide({event_score}, pow(2, least({max_pow}, divide({event_age_hours}, {event_halflife_hours}))))))"
+    issue_age_hours = "divide(now() - min(timestamp), 3600)"
+    issue_score = 1
+    issue_halflife_hours = "multiply(24, 7)"  # issues half in value every week
```

Comment:
Nit: Might make sense to just calculate the number of hours in python?

Reply:
I think I tried that and it was upset - let me see again.